sdk/multisig/src/instructions/multisigAddSpendingLimit.ts

```diff
@@ -47,6 +47,7 @@ export function multisigAddSpendingLimit({
         createKey,
         vaultIndex,
         mint,
+        // @ts-ignore
```

Comment:
@vovacodes This ts-ignore was not needed before, did that function work normally before ?